File: spring-skill/src/main/java/com/stackroute/educationservice/springskill/domain/Section.java
package com.stackroute.educationservice.springskill.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Section {
    String sectionId;
    String userId;
    String operationType;
    Chicklet chicklets[];
}

File: angular/src/app/cards/experience-class/section.ts
import { Chicklets } from "./chicklets";
export class ExperienceSection {
    constructor(private sectionId: string,
        private userId: string,
        private operationType: string,
        private chicklets: Chicklets[]) {
        this.sectionId = sectionId;
        this.userId = userId;
        this.operationType = operationType;
        this.chicklets = chicklets;
    }
}